homestar-runtime/tests/network/gossip.rs

```diff
@@ -25,21 +29,56 @@ const SUBSCRIBE_NETWORK_EVENTS_ENDPOINT: &str = "subscribe_network_events";
 const UNSUBSCRIBE_NETWORK_EVENTS_ENDPOINT: &str = "unsubscribe_network_events";
 
 #[test]
+#[named]
```

Comment:
@bgins had the right idea that naming itself doesn't matter, just uniqueness. We can use the UUID lib as a dep-dependency and name it with that.